### ISSUE TITLE
eddystone_url.ino   Does not see the program Physical WEB and Nearby

### ISSUE THREAD

**Issue by ESAlexhi** [floe/BTLE#21]
I'm testing   eddystone_url.ino   and  beacon(nrF24L01)   does not see the program Physical WEB and Nearby. The Program  nrfConnect  sees  beacon as Physical Web Beacon What can be wrong?

**Comment by floe** [comment]
OK, so if nRF Connect can detect it, then Nearby is maybe applying some extra filtering (e.g. what URLs are allowed)? Does the nRF app show the correct URL?

**Comment by ESAlexhi** [comment]
Yes nRF 
![ble](https://user-images.githubusercontent.com/10528761/42390983-77e88486-8156-11e8-8633-7f954aebfc59.jpg)
 shows the correct address

**Comment by ESAlexhi** [comment]
If I press OPEN I get to the site  https://qr.net/fNA8YZ    , but   the program Physical WEB and Nearby not see Beacon .

**Comment by floe** [comment]
Have you tried https://github.com/google/eddystone/tree/master/tools/eddystone-validator ?

**Comment by ESAlexhi** [comment]
Yes  , not see :(       Beacon see only the program  nRF Connect

**Comment by floe** [comment]
Not really sure what else the reason could be, maybe @ReDetection has an idea?

**Comment by ReDetection** [comment]
No ideas so far. I did tests using the program Physical WEB too, so I'm pretty sure it should work. I haven't seen Nearby tool in android I have around, is it builtin? How to find it? Or it will just present notification under certain condition?

**Comment by ReDetection** [comment]
I recently found a person with a device which has Nearby app and I can confirm beacon didn't appear there (though it still shows in Physical WEB app and nRF Connect. Also I discovered a little [piece of documentation](https://developers.google.com/nearby/notifications/get-started#2-configure-your-beacons) of google, so now I wonder if I have to register every beacon URL in order to work with Nearby app?

**Comment by floe** [comment]
Hm, the way I understand it, that specifically does not apply to Eddystone-URL beacons. For -UID and -EID, you need Google support, but (theoretically) not for -URL. Maybe it's the random MAC address?

**Comment by ReDetection** [comment]
😞 
https://android-developers.googleblog.com/2018/10/discontinuing-support-for-android.html
